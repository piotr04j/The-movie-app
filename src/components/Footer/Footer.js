import React from 'react';


const Footer = () => {
  return (
    <div className="footer bg-primary">
      <div className="container">
        <img src="/images/logo.png" alt="API Logo" className="img-fluid" />
        <p>This product uses the TMDb API but is not endorsed or certified by TMDb</p>
      </div>
    </div>
  )
}

export default Footer;
