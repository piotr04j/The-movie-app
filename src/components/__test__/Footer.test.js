import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../Footer/Footer';

it('render correctly', () => {
    const tree = renderer.create(<Footer />).toJSON(); 
    expect(tree).toMatchSnapshot();
});