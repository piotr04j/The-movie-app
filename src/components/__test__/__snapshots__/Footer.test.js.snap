// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  className="footer bg-primary"
>
  <div
    className="container"
  >
    <img
      alt="API Logo"
      className="img-fluid"
      src="/images/logo.png"
    />
    <p>
      This product uses the TMDb API but is not endorsed or certified by TMDb
    </p>
  </div>
</div>
`;
